Proiect/new-evaluation.css

```diff
@@ -67,5 +80,64 @@
 		}
 
 		li {
-			display: inline;
+			float: left;
+			text-align: left;
+			width: 16%;
+			margin: 2px;
+			min-height: 36px;
+			font-family: Arial;
+			font-size: 15px;
+		}
+
+		#submit {
+			float: right;
+			width: 150px;
+			height: 50px;
+			background-color: #419fe2;
+			color: white;
+			border: none;
+			border-radius: 2px;
+			font-size: 20px;
+			box-shadow: 0px 3px 4px grey;
```

Comment:
From what I know, percentage values don't work on box-shadows. I looked it up online now, but wasn't able to find anything helpful. Maybe you could enlighten me? :))